std/typecons.d

```diff
@@ -4628,6 +4640,63 @@ unittest
     //func(n);
 }
 
+///The proxied value must be an $(B lvalue).
+unittest
+{
+    struct NewIntType
+    {
+        //Won't work; the literal '1' is
+        //is an rvalue, not an lvalue
+        //mixin Proxy!1; 
+        
+        //Okay, n is an lvalue
+        int n;
+        mixin Proxy!n;
+        
+        this(int n) { this.n = n; }
+    }
+    
+    NewIntType nit = 0;
+    nit++;
+    assert(nit == 1);
+    
+    
+    struct NewObjectType
+    {
+        //Ok, "new Object" is an lvalue.
+        //This code is a bit weird, though.
+        mixin Proxy!(new Object);
```

Comment:
To demonstrate that *any* lvalue can be used with Proxy, even if it doesn't make sense.